src/util/globals.js

```diff
@@ -1,12 +1,14 @@
 import fetch from 'node-fetch';
 
+import config from './config';
+
 // hello polyfills
 global.URL = require('url').URL;
 global.fetch = fetch;
 
 // runtime values needed by browser and server
 global.CONFIG = {
-	ASSET_SERVER_HOST : process.env.ASSET_SERVER_HOST,
-	ASSET_SERVER_PORT : process.env.ASSET_SERVER_PORT,
+	ASSET_SERVER_HOST : config.get('asset_server.host'),
+	ASSET_SERVER_PORT : config.get('asset_server.port'),
 };
```

Comment:
pretty sure this `global` object is dead now - I've been meaning to delete it for aaaages.